### PR TITLE
DecisionTree plotter

### PR DESCRIPTION
**Pull request by ariguiba**
Added an example for how to plot a DecisionTreeClassifier object from the DecisionTree package. The example is inspired form the syntaxtree.jl example and presents a straightforward way to plot a DecisionTreeClassifier with the feature values in the nodes.

### CODE REVIEW

**Comment by hexaeder** [comment]
Hey, thanks for the contribution! 
I've added MLJ to the docs environment and linked the example withing `docs/make.jl`. However, `load_data` seems to be undefined. Please try to build the docs locally by either
- using the `localmake.jl` script (needs LiveServer.jl in your global env)
- activating `docs` env and run `docs/make.jl`

File: docs/examples/decisiontree.jl [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+In this example we are going to plot a decision tree of type [`DecisionTree.jl`](https://github.com/JuliaAI/DecisionTree.jl) using the
+Bucheim Layout from [`NetworkLayout.jl`](https://github.com/JuliaGraphs/NetworkLayout.jl).
+=#
+using CairoMakie
+using Graphs
+using GraphMakie
+using MLJ
+using NetworkLayout
+
```
Comment (behinger) [inline docs/examples/decisiontree.jl:12]: `using DecisionTree`
is missing here (has `load_data`)

**Review by behinger — COMMENTED** [review]
I'm a co-author of this code :)

File: docs/examples/decisiontree.jl (hunk 1 above)
Comment (hexaeder) [reply docs/examples/decisiontree.jl:12]: ... then `DecisionTree` should go in the `docs/Project.tml` as well

BTW: sorry for ignoring you on Discord again, I don't get any notifications there. I think its allways easier to reach me via discourse PM, mail or on slack. But to answer the qustion: The recipe style isfine, those are examples anyway so I think it is nice to showcase various different ways of working with Makie/GraphMakie.

**Comment by behinger** [comment]
Should we merge - or did I miss something?

**Comment by hexaeder** [comment]
I am on Vacation and didn't built the docs locally yet to look at them, I hope to do that soon. Nothing to do at your and right now, thanks for fixing the CI issues!

**Pull request merged**